--- a/src/prompt-box.vala	2015-03-25 02:19:42.000000000 +0100
+++ b/src/prompt-box.vala	2016-09-03 17:45:27.360812129 +0200
@@ -229,7 +229,7 @@
 
         small_name_label = new FadingLabel ("");
         small_name_label.override_font (Pango.FontDescription.from_string ("Ubuntu 13"));
-        small_name_label.override_color (Gtk.StateFlags.NORMAL, { 1.0f, 1.0f, 1.0f, 1.0f });
+        small_name_label.override_color (Gtk.StateFlags.NORMAL, { 0.0f, 0.0f, 0.0f, 1.0f });
         small_name_label.yalign = 0.5f;
         small_name_label.xalign = 0.0f;
         small_name_label.margin_left = 2;
